runtime/pack/dist/opt/netrw/autoload/netrw.vim

```diff
@@ -1534,7 +1534,7 @@ fun! NetrwStatusLine()
   else
```

Comment:
(Sorry cannot comment on non-changed lines).
the version in `plugin` is a bit cleaner by using: 
```
unlet! w:netrw_explore_bufnr w:netrw_explore_line
```
That is more readable than the version used here with the `if .. | ... |endif`.